### ISSUE TITLE
React / DOMException: Failed to execute importScripts on WorkerGlobalScope

### ISSUE THREAD

**Issue by Gishan86** [naptha/tesseract.js#643]
**Describe the bug**
I'm trying to use tesseract.js version 3.0.1 within my React app by following this simple tutorial: https://www.twilio.com/blog/tesseract-js-react-ocr-part-one

Always crashes at getCore.js:20 with:
```
Uncaught (in promise) DOMException:
Failed to execute 'importScripts' on 'WorkerGlobalScope':
The script at 'http://localhost:3000/null' failed to load.

```

I've already tried alternative methods to the worker that is described in the tutorial (ie "Tesseract.recognize") but this also leads to the same crash.

While debugging I saw that in getCore.js the variable "corePath" is set to "http://localhost:3000/null". Therefore it's directly used in line 20 which leads to the crash.

But why is it set like this?
Do I have to configure somthing that's not mentioned in the tutorial?

**To Reproduce**
1. Simply create a new React app
2. Copy & Paste code from Tutorial
3. Upload local image
4. Process image with Tesseract

**Desktop (please complete the following information):**
 - OS: Windows 10 (latest updates)
 - Browser: Chome 104.0.5112.102 (latest)
 - Tesseract Version: 3.0.1
 - Node Version: 16.16.0

**Comment by Balearica** [comment]
Thanks for reporting this.  I'm guessing that the way we create a default `corePath` was broken in the last release, and all of the tests manually specify a `corePath` so this was not flagged.  Probably not hard to fix so I'll try and release a patch shortly.

**Comment by Balearica** [comment]
@Gishan86 I pushed a patch and created a new release--please try with version `3.0.2` and confirm whether or not the problem remains.

**Comment by Gishan86** [comment]
@Balearica Problem is resolved with 3.0.2
Thanks a lot!
